confluent_kafka/avro/serializer/Util.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
Module names shouldn't be capitalized (i.e. the file name should just be `util.py`, not `Util.py`. In fact, this is true for all the filenames under this module (CachedSchemaRegistryClient, AvroProducer, etc).